examples/serial.c

```diff
@@ -0,0 +1,41 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdint.h>
+#include <pthread.h>
+
+#include "wiringX.h"
+
+typedef struct wiringXSerial_t {
+	unsigned int baud;
+	unsigned int databits;
+	unsigned int parity;
+	unsigned int stopbits;
+	unsigned int flowcontrol;
+} wiringXSerial_t;
```

Comment:
Yes. But without this it reports error when compiling 'variable ‘wiringXSerial’ has initializer but incomplete type' .
I didn't figure out why this struct definition in wiringx.h can't be found in serial.c when the head file is included. So I add this and it works.